src/components/Documentation/LeftNav.js

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import { PALETTE } from "constants/styles";
+
+import { buildRelPath } from "helpers/documentation";
+
+import { Link } from "basics/Links";
+import { ListItem } from "basics/Text";
+
+import Articles from "components/Documentation/Articles";
+import {
+  NavAbsoluteEl,
+  AbsoluteNavFooterEl,
+  SideNavContainer,
+} from "components/Navigation/SharedStyles";
+
+const TopicsEl = styled.ul`
+  list-style-type: none;
+  padding: 0;
+  padding-bottom: 1rem;
+  margin-right: 1rem;
+`;
+const RootItemEl = styled(ListItem)`
+  padding: 0.5rem 0;
+`;
+const LinkEl = styled(Link)`
+  color: ${PALETTE.purpleBlue};
+`;
+const RootLinkEl = styled(Link)`
+  color: ${PALETTE.black80};
+  line-height: 1.75rem;
+  text-decoration: none;
+`;
+
+export const LeftNav = ({
```

Comment:
I yanked this out of `templates/Documentation` because it was getting large, and I needed to use it in a second location.